Engine/DataFeeds/SubscriptionSynchronizer.cs

```diff
@@ -126,7 +135,7 @@ public TimeSlice Sync(IEnumerable<Subscription> subscriptions)
                         }
                     }
 
-                    if (packet.Count > 0)
+                    if (packet != null && packet.Count > 0)
```

Comment:
style - no need to change - using the null navigation operation => `if (package?.Count > 0)` should work as well.

Reply:
Will do 👍